modules/oxide/src/less/theme/components/tinymceai/spinner.less

```
@@ -26,4 +31,12 @@
       &:nth-child(2) { animation-delay: -.16s; }
     }
   }
+
+  // Make the dots smaller to fix small 16x16 spinner container
+  .tox-ai__spinner--dots.tox-ai__spinner--small {
```

Stylelint (17.2.0)

[error] Expected class selector ".tox-ai__spinner--dots" to be kebab-case (selector-class-pattern)

(selector-class-pattern)